pkgs/development/libraries/qt-5/modules/qtbase.nix

```
@@ -162,6 +162,8 @@ stdenv.mkDerivation {
   qtQmlPrefix = "lib/qt-${qtCompatVersion}/qml";
   qtDocPrefix = "share/doc/qt-${qtCompatVersion}";
 
+  dontStrip = developerBuild;
```

```suggestion
  # don't strip away debugging symbols when build with developerBuild
  dontStrip = developerBuild;
```

I think we should explain this because generally we strip everything.